models/core/logistic_regression.py

```diff
@@ -0,0 +1,115 @@
+from sklearn import linear_model
+import pandas as pd
+import joblib
```

Comment:
Why is this needed? Is it possible to simply load in the LogisticRegression model via sci-kit learn's interface?

Reply:
[sklearn documentation](https://scikit-learn.org/stable/model_persistence.html) recommends joblib.